src/CopyrighFixer/CopyrighFixer/owner.cpp

```diff
@@ -0,0 +1,24 @@
+#include "owner.h"
```

Comment:
```suggestion
//#
//# Copyright (C) 2021-2021 QuasarApp.
//# Distributed under the lgplv3 software license, see the accompanying
//# Everyone is permitted to copy and distribute verbatim copies
//# of this license document, but changing it is not allowed.
//#

#include "owner.h"
```
you forget to add our copyright )